packages/js/src/ai-assessment-fixes/components/ai-assessment-fixes-button.js

```diff
@@ -24,23 +24,44 @@ import { ReactComponent as LockClosed } from "../../../images/lock-closed.svg";
  */
 const AIAssessmentFixesButton = ( { id, isPremium } ) => {
 	const aiFixesId = id + "AIFixes";
-	const ariaLabel = __( "Optimize with AI", "wordpress-seo" );
 	const [ isModalOpen, , , setIsModalOpenTrue, setIsModalOpenFalse ] = useToggleState( false );
 	const activeAIButtonId = useSelect( select => select( "yoast-seo/editor" ).getActiveAIFixesButton(), [] );
 	const activeMarker = useSelect( select => select( "yoast-seo/editor" ).getActiveMarker(), [] );
 	const { setActiveAIFixesButton, setActiveMarker, setMarkerPauseStatus } = useDispatch( "yoast-seo/editor" );
 	const focusElementRef = useRef( null );
-	const [ buttonClass, setButtonClass ] = useState( "" );
+	const [ tooltipClass, setTooltipClass ] = useState( "" );
+
+	const defaultLabel = __( "Optimize with AI", "wordpress-seo" );
+	const tooLongLabel = __( "Your text is too long for the AI model to process.", "wordpress-seo" );
+	const htmlLabel = __( "Please switch to the visual editor to use AI.", "wordpress-seo" );
```

Comment:
The `tooLongLabel` and `htmlLabel` are new. The former has been approved but the latter has not yet been confirmed 😸

Reply:
I guess, even if the `htmlLabel` needs to be adjusted it's a small change that we can make in a tiny PR later on. 🙂

Reply:
@mhkuu This isn't wrong. I just thought, to stay in line with "Optimize with AI," we might consider saying: 
`Please switch to the visual editor to optimize with AI.`

Reply:
Nice, implemented! 🎉